backup_and_restore/application_backup_and_restore/troubleshooting.adoc

```diff
@@ -27,6 +27,7 @@ include::modules/velero-obtaining-by-accessing-binary.adoc[leveloffset=+1]
 
 include::modules/oadp-debugging-oc-cli.adoc[leveloffset=+1]
 include::modules/migration-debugging-velero-resources.adoc[leveloffset=+1]
+include::modules/migration-debugging-velero-admin-webhooks.adoc[leveloffset=+1]
```

Comment:
@RichardHoch 
Add 'Additional resources' after this line - see my comment below.